vignettes/protein_structure_workflow.Rmd

```diff
@@ -0,0 +1,195 @@
+---
+title: "Protein Structure Analysis Workflow"
+author: "Jan-Philipp Quast"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Protein Structure Analysis Workflow}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+<style>
+body {
+text-align: justify}
+</style>
+
+```{r, include = FALSE}
+test_protti <- identical(Sys.getenv("TEST_PROTTI"), "true")
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+This vignette will take you through all important tools **protti** provides for structural data analysis. These include functions that allow you to fetch information about structures both on a general and on the atom level. Additionally, distances between atoms can be extracted from structures. Lastly, B-factors* of structures can be manipulated to contain custom values, which gives you the ability to colour regions on structures as well as apply different values associated with each amino acid as a colour gradient. 
+
+_*B-factors or temperature factors are used in structural biology to describe the atomic displacement in the structure. In low resolution structures they might correspond to the flexibility of regions, while in high resolution structures it should not be used to infer information regarding flexibility._
+
+In structural proteomics, protein strucural changes can be determined using methods such as limited proteolysis coupled to mass spectrometry ([LiP-MS](https://www.nature.com/articles/nbt.2999)). In case of LiP-MS, structural changes can be identified with peptide or precursor* centric proteomics in the form of peptide fold changes. Relating this information to positions within the 3D space of a protein can be very helpful for inferring potential functional roles of those structural changes. If structural changes occur close to the active site of an enzyme, kinetics might be affected, if they occur at a protein-protein binding interface, the corresponding protein complex might have dissociated or associated. 
```

Comment:
In the first sentence I would mention also other techniques, such as XL-MS or HDX-MS. It sounds like you want to mention multiple techniques but actually only list LiP-MS